bin/v-add-web-domain-backend

```diff
@@ -86,9 +86,9 @@ fi
 #                       Hestia                             #
 #----------------------------------------------------------#
 
-# Restart backend server
+# Restart php interpeter
```

Comment:
s/interpeter/interpreter/ here and all other occurrences

Reply:
Resolved, thanks for pointing this out.